README.md

```diff
@@ -11,15 +11,15 @@ https://licensebuttons.net/l/by/4.0/88x31.png
 https://creativecommons.org/licenses/by/4.0/
 )
 
-This repository contains the specifications of the Asset Administration Shell, including the schemas of the serializations, the rules applied, how the specification is mapped into the serializations, and examples of how to use the schemas.
+This repository contains specifications of the Asset Administration Shell, including the normative schemas of the serializations, the rules applied to create them, how the specification is mapped into the serializations, and examples of how to use the schemas.
 
 ## Schemas
 
-We provide schemas of the Asset Administration Shell for JSON, RDF and XML. 
+The schemas of the Asset Administration Shell for JSON, RDF and XML are provided in the respective directories. 
```

Comment:
Additionally the xmi file of the UML modeling is provided.
What about YAML?

Reply:
I added statements for YAML and XMI